contracts/services/PayoutManager.sol

```diff
@@ -101,68 +81,78 @@ contract PayoutManager {
         return assetPayouts;
     }
 
-    function hasClaimedFunds(uint256 _payoutId, address _wallet) public view payoutExists(_payoutId) returns (bool) {
+    function getPayoutIdsForOwner(address _ownerAddress) public view override returns (uint256[] memory) {
+        return payoutsByOwnerAddress[_ownerAddress];
+    }
+
+    function getPayoutsForOwner(address _ownerAddress) public view override returns (Payout[] memory) {
+        uint256[] memory payoutIds = payoutsByOwnerAddress[_ownerAddress];
+        Payout[] memory ownerPayouts = new Payout[](payoutIds.length);
+
+        for (uint i = 0; i < payoutIds.length; i++) {
+            ownerPayouts[i] = payoutsById[payoutIds[i]];
+        }
+
+        return ownerPayouts;
+    }
+
+    function getAmountOfClaimedFunds(uint256 _payoutId, address _wallet) public view override payoutExists(_payoutId) returns (uint256) {
         return payoutClaims[_payoutId][_wallet];
     }
 
+    function hasClaimedFunds(uint256 _payoutId, address _wallet) public view override payoutExists(_payoutId) returns (bool) {
```

Comment:
this is a redundant function in my opinion. if someone is interested in knowing if user has claimed funds for a payout they will check if
```payoutManager.getAmountOfClaimedFunds(payoutId, investor) > 0```
Do you agree @domagojlatecki @githinho  ?

Reply:
Yes, let's remove this to have a simpler contract.